frontend/common/kubeflow-common-lib/package.json

```diff
@@ -81,6 +81,9 @@
     "http-proxy-middleware": "2.0.7",
     "webpack-dev-middleware": "6.1.2",
     "undici": "6.21.1",
-    "webpack": "5.94.0"
+    "webpack": "5.94.0",
+    "@babel/runtime": "7.26.10",
+    "esbuild": "0.25.1",
+    "vite": "5.4"
```

Comment:
Why need to add those 3? 
I see they are in the jupyter package.json and overwritten but are they needed here too?

Reply:
fixes the remaining vulns in the common lib. 
Since they are the same as the ones coming from jupyter, I figured might as well fix them here as well.